components/inputs/input-number.js

```diff
@@ -363,6 +365,7 @@ class InputNumber extends FocusMixin(LabelledMixin(SkeletonMixin(FormElementMixi
 					<slot slot="left" name="left"></slot>
 					<slot slot="right" name="right"></slot>
 					<slot slot="after" name="after"></slot>
+					${this._renderInlineHelpNested(this._inlineHelpId)}
```

Comment:
Is `_renderInlineHelpNested` needed? I was thinking that all we would need here is:
```javascript
<slot slot="inline-help" name="inline-help"></slot>
```
In such case, I don't think these components need to extend `InputInlineHelpMixin`. The slot change handler and styles will already be taken care of by `d2l-input-text`. Generating the id is already handled too.